control/cli.py

```diff
@@ -124,7 +124,9 @@ def stub(self):
 
     def connect(self, host, port, client_key, client_cert, server_cert):
         """Connects to server and sets stub."""
-        server = "{}:{}".format(host, port)
+        if ":" in host and not host.strip().startswith("["):
```

Comment:
nit: we encounter this condition repeatedly; it might be worthwhile to abstract it into a function like `ipv6_addr` that returns a boolean value

Reply:
Moved all this code to a separate function:
```
    def escape_address_if_ipv6(addr) -> str:
        ret_addr = addr
        if ":" in addr and not addr.strip().startswith("["):
            ret_addr = f"[{addr}]"
        return ret_addr
```